data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+		log `He Yi asked me to sell him my cloaking device. I refused.`
+
+
+
+ship "Shield Beetle" "Enhanced Shield Beetle (Pulse)"
+	add attributes
+		"outfit space" 2
+		"weapon capacity" 3
+	outfits
+		"Tripulse Shredder" 8
+		"Boulder Reactor"
+		"Geode Reactor"
+		"Hai Ravine Batteries"
+		"Hai Diamond Regenerator" 2
+		"Hai Williwaw Cooling" 3
+		"Outfits Expansion"
+		"Pulse Rifle" 40
+		"Dual Pulse Pistols"
+		`"Bondir" Atomic Thruster`
+		`"Bufaer" Atomic Steering`
+		"Hyperdrive"
+
+ship "Shield Beetle Prototype" "Shield Beetle Prototype (Black)"
+	sprite "ship/hai marauder shield beetle"
+	thumbnail "thumbnail/hai marauder shield beetle"
+	add attributes
+		"hull" 1000
+		"spinal mount" 1
+		"cloak" .01
+		"cloaking energy" 10
+		"cloaking fuel" .5
+		"cloaking heat" 10
+		"engine capacity" 5
+		"outfit space" 2
+		"weapon capacity" -1
+	outfits
+		"Sunbeam" 4
+		"Flamethrower" 2
+		"Korath Detainer"
+		"Korath Grab-Strike"
+		"Korath Banisher"
+		"Bullfrog Anti-Missile" 3
+		"Large Heat Shunt"
+		"Systems Core (Large)" 2
+		"Outfits Expansion" 4
+		"Jump Drive"
+		"Fuel Processor"
+		"Wanderer Ramscoop"
+		`"Bufaer" Atomic Steering`
+		"Thruster (Planetary Class)"
+		"White Sun Reactor"
+		"Generator (Furnace Class)"
+		"Korath Repeater Rifle" 20
+		"Pulse Rifle" 60
+		"Dual Pulse Pistols" 10
+	gun 0 -7 "Korath Detainer"
+	gun -22 -102 "Sunbeam"
+	gun 22 -102 "Sunbeam"
+	gun -34 -68 "Sunbeam"
+	gun 34 -68 "Sunbeam"
+	gun -49 -38
+	gun 49 -38
+	gun -61 -14 "Flamethrower"
+	gun 61 -14 "Flamethrower"
+	turret -51 21.5 "Korath Grab-Strike"
+	turret 51 21.5 "Bullfrog Anti-Missile"
+	turret -56.5 75 "Korath Banisher"
+	turret 56.5 75 "Bullfrog Anti-Missile"
+	turret 0 85.5 "Bullfrog Anti-Missile"
+
+mission "Unfettered: Start Of The War"
+	name `Strength Of A Leader`
+	description `Prove your strength against Si-Ra and Hi-Ye in a battle, but do not kill them.`
+	source "Darkcloak"
+	substitutions
+		<question> "How does your ship cloak?"
+			not "Unfettered: Cloaking Interest: done"
+		<question> "Why is your hull black?"
+			has "Unfettered: Cloaking Interest: done"
+	to offer
+		has "event: Unfettered: shield beetle mass production"
+		has "event: Unfettered: new cooling system"
+		has "Unfettered: Special Cargo 2: done"
+		has "main plot completed"
+		has "language: Wanderer"
+	on offer
+		conversation
+			branch "came alone"
+				"ships: Transport" * 6 + "ships: Light Freighter" * 3 + "ships: Heavy Freighter" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 4 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 <= 18
+			`You find Si-Ra and Hi-Ye waiting for you on the spaceport. Si-Ra starts speaking to you, "You have brought too many ships, come back with only your flagship, or a few lighter ships."`
+				defer
+			label "came alone"
+			`You find Si-Ra and Hi-Ye waiting for you on the spaceport. Si-Ra asks, "Are you ready for war, <last>?"`
+			choice
+				`	"Not yet. I will come back when I am."`
+					goto later
```

Comment:
Let's move the option to `defer` later in the dialogue once we actually know what's going on. The player will probably make the decision about whether they want to defer after they know what they're up against.